aws-gitlab-main.yml

```diff
@@ -59,33 +59,67 @@ Parameters:
     Description: CidrBlock for RDS Security Group
   
   #DATABASE
-  DBInstanceClassParameter:
+  GitLabDBInstanceClassParameter:
     Type: String
     Description: DB EC2 Instnace Class. Default is db.m4.large
     Default: db.m4.large  
 
-  DBInstanceIdentifierParameter:
+  GitLabDBInstanceIdentifierParameter:
     Type: String
     Description: Name for the DB instance. 
-  
-  GitLabDBMasterUsernameParameter:
-    Type: String
-    Description: Master user name for the DB instance
-  
-  GitLabDBMasterUserPasswordParameter:
-    Type: String
-    Description: Password for the master user. At least 8 characters. No slash, at symbol, or parentheses. 
 
   GitLabAllocatedStorageParameter:
     Type: String
     Description: Amount of storage in gigabytes to be allocated for the database intsance
     Default: '100' #minimum for PostgreSQL 
 
   #ELASTICACHE/REDIS
-  CacheNodeTypeParameter:
+  GitLabCacheNodeTypeParameter:
     Type: String
     Description: Node type;  Multi-AZ not supported on T1 instances
     Default: cache.t3.medium 
+  
+  #BASTION HOSTS
+  GitLabBastionSecurityGroupCIDRParameter:
+    Type: String
+    Description: Enable SSH access from what IP address(es). For stricter security, specify a single private IP 
+    Default: 127.0.0.1/8 #loopback
+
+  BastionHostAAvailabilityZoneParameter:
+    Type: AWS::EC2::AvailabilityZone::Name
+    Description: AZ for Bastion Host A
+    
+  GitLabBastionHostAInstanceTypeParameter:
+    Type: String
+    Description: EC2 instance type for Bastion Host A
+    Default: t2.micro 
+
+  GitLabBastionHostAAMIParameter:
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Description: Latest AMI for Bastion Host A
+    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2 #Ubuntu-Server-18.04-LTS-HVM
```

Comment:
this is clearly variable path to amzn2 instance not ubuntu... make sure to put correct param here

Reply:
oops :(